app/javascript/styles/components/Spotlights.scss

```diff
@@ -136,82 +119,50 @@
   }
 
   &__info {
-    @include media(large) {
-      grid-column: 2 / 4;
-      -ms-grid-column-span: 3;
-    }
-    @include media(medium) {
-      grid-column: 2 / 3;
-      -ms-grid-column-span: 2;
+    display: inline-block;
+    max-width: 100%;
+    padding: 0 0 0 4rem;
+    vertical-align: top;
+    width: 54rem;
+
+    @media screen and (max-width: 1350px) {
+      padding: 0 1rem;
+      width: 100%;
     }
-    grid-column: 2 / 5;
-    -ms-grid-column-span: 4;
   }
 }
 
 .gallery-spotlight {
-  @include media(medium) {
-    grid-template-columns: repeat(4, 1fr);
-    -ms-grid-columns: 1fr 1fr 1fr 1fr;
-  }
-  @include media(small) {
-    grid-template-columns: repeat(3, 1fr);
-    -ms-grid-columns: 1fr 1fr 1fr;
-  }
   column-gap: 40px;
-  display: grid;
-  grid-template-columns: repeat(5, 1fr);
-  display: -ms-grid;
-  -ms-grid-columns: 1fr 1fr 1fr 1fr 1fr;
+  margin: 0 auto;
 
   &__title {
     margin-left: -20px;
   }
 
   &__image {
+    display: inline-block;
```

Comment:
Nice refactor all around here; much simpler and easier to read than my previous Media Query Stew. :)